public/locales/en/translation.json

```diff
@@ -44,18 +44,54 @@
         "id": 0
       },
       {
-        "title": "Web Developer",
+        "title": "Web developer",
         "name": "Dameon Peterson",
         "description": "Certe, inquam pertinax non quo quaerimus, non fuisse torquem detraxit hosti et quidem exercitus quid ex eo ortum, tam crudelis fuisse, nihil molestiae consequatur, vel illum, qui officia deserunt mollitia animi, id omnia referri oporteat.",
         "address": "Istanbul, Turkey",
         "phoneNumber": "+906467291373",
         "date": "27/10/2020",
         "id": 1
+      },
+      {
+        "title": "Translator",
+        "name": "Dameon Peterson",
+        "description": "Certe, inquam pertinax non quo quaerimus, non fuisse torquem detraxit hosti et quidem exercitus quid ex eo ortum, tam crudelis fuisse, nihil molestiae consequatur, vel illum, qui officia deserunt mollitia animi, id omnia referri oporteat.",
+        "address": "Istanbul, Turkey",
+        "phoneNumber": "+906467291373",
+        "date": "27/10/2020",
+        "id": 2
+      },
+      {
+        "title": "Architect",
+        "name": "Dameon Peterson",
+        "description": "Certe, inquam pertinax non quo quaerimus, non fuisse torquem detraxit hosti et quidem exercitus quid ex eo ortum, tam crudelis fuisse, nihil molestiae consequatur, vel illum, qui officia deserunt mollitia animi, id omnia referri oporteat.",
+        "address": "Istanbul, Turkey",
+        "phoneNumber": "+906467291373",
+        "date": "27/10/2020",
+        "id": 3
+      },
+      {
+        "title": "HR manager",
+        "name": "Dameon Peterson",
+        "description": "Certe, inquam pertinax non quo quaerimus, non fuisse torquem detraxit hosti et quidem exercitus quid ex eo ortum, tam crudelis fuisse, nihil molestiae consequatur, vel illum, qui officia deserunt mollitia animi, id omnia referri oporteat.",
+        "address": "Istanbul, Turkey",
+        "phoneNumber": "+906467291373",
+        "date": "27/10/2020",
+        "id": 4
+      },
+      {
+        "title": "Teacher",
+        "name": "Dameon Peterson",
+        "description": "Certe, inquam pertinax non quo quaerimus, non fuisse torquem detraxit hosti et quidem exercitus quid ex eo ortum, tam crudelis fuisse, nihil molestiae consequatur, vel illum, qui officia deserunt mollitia animi, id omnia referri oporteat.",
+        "address": "Istanbul, Turkey",
+        "phoneNumber": "+906467291373",
+        "date": "27/10/2020",
```

Comment:
what is this date?

Reply:
It is dummy data for the date of posting a job, I just duplicated the data which was already there.

Reply:
key name needs to be improved. startdate? posting date? end date of posting?